docs/core/deploying/trim-warnings/il2055.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "IL2055: Call to 'System.Type.MakeGenericType' cannot be statically analyzed by the trimmer"
+description: "Learn about trim warning IL2055: MakeGenericTypeCall"
+ms.date: 07/23/2021
+ms.topic: reference
+author: mateoatr
+ms.author: matorre
+f1_keywords:
+  - "IL2055"
+---
+# IL2055: Call to 'System.Type.MakeGenericType' cannot be statically analyzed by the trimmer
+
+## Cause
+
+Calls to <xref:System.Type.MakeGenericType> cannot be statically analyzed by the trimmer.
```

Comment:
```suggestion
Call to <xref:System.Type.MakeGenericType> cannot be statically analyzed by the trimmer.
```

Trimmer can deal with lot of MakeGenericType calls, just not the one the warnings is reproted on.